### PR TITLE
Columns toggler

### LINKED ISSUE #133: Columns toggler
Table view (added in #132) should allow user to decide which columns are visible

### PR DESCRIPTION
**Pull request by smogg**
Implements toggler (closes #133):

screen shot 2016-05-10 at 16 04 42

### CODE REVIEW

**Comment by lvh** [comment]
Please merge master forward

File: src/desdemona/ui/table.cljs [hunk 1]
```diff
@@ -2,10 +2,44 @@
   (:require [wilson.dom :as wd]
             [reagent.session :as session]))
 
+(defn toggle-key
+  "Adds/removes k in given coll and returns the coll."
+  [coll k]
+  (if (contains? coll k)
```
Comment (lvh) [inline src/desdemona/ui/table.cljs]: Didn't we decide that this is `(some? #{k} coll)`?

Also you can just pick the fn with this and then call coll k on it, which limits repetition.

**Comment by lvh** [comment]
This LGTM modulo comment and looks _awesome_. I would suggest maybe looking at what dommy or something can offer to see if it can make the tests better.

**Comment by smogg** [comment]
Dommy doesn't really give us much more (as far as I'm aware) apart from a little easier to read selections. I'll add it anyway since we're gonna use it a lot in the future

**Pull request merged**